build.gradle

```diff
@@ -15,6 +15,9 @@ repositories {
 
 dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation("org.springframework.boot:spring-boot-starter-jdbc")
```

Comment:
maybe change it to `implementation 'org.springframework.boot:spring-boot-starter-jdbc'` to be consistent with other dependency declarations

Reply:
I made it double quote since most of them is like that